src/visualize/Visualizer.cpp

```diff
@@ -60,6 +60,14 @@ void Visualizer::WriteVtuFile(std::string filename, bool asBinary)
     mGrid.ExportVtuDataFile(filename, asBinary);
 }
 
+void Visualizer::SetCells(const Group<CellInterface>& cells)
+{
+    mCells = cells;
+    mPointIds.clear();
+    mGrid = UnstructuredGrid();
+    WriteGeometry();
+}
```

Comment:
So basically you construct the Visualizer again. Only exception is the handler which stays.
Wouldn't it be more natural to do just that: Construct a new Visualizer?

Reply:
That is actually a good idea. So instead of 
~~~
// instead of
mVisualizer.SetCells(newCells);
// I could do
mVisualizer = Visualizer(newCells, VoronoiHandler(...));
~~~
The first one is more convenient. But the second one does not require `SetCells()`. I'll think about that.